README.rst

```diff
@@ -352,6 +352,13 @@ The list of IPv6 CIDR blocks of the subnet in which the interface resides, e.g.
 ``['2001:db8:abcd:ef00::/64']``. If the subnet does not have any IPv6 CIDR
 blocks or the instance isn't in a VPC, the list will be empty, e.g. ``[]``.
 
+``tags: map``
```

Comment:
`map` is not a real Python type hint, what we want is the same as the source:

```suggestion
``tags: dict[str, str]``
```

Reply:
Changing this as making the value a lazy mapping

Also this documentation was in the wrong place - it was inside the `NetworkInterface` docs!